src/1_api.js

```diff
@@ -51,12 +51,32 @@ Server.prototype.serializeObject = function(obj, prefix) {
  * @param {Object.<string, *>} data
  */
 Server.prototype.getUrl = function(resource, data) {
-	var k, v, err;
-	var url = resource.destination + resource.endpoint;
+	var k, v, err,
```

Comment:
Goo call - honestly, the entire getUrl method could probably be refactored at some point. I haven't done much with it since I took this over.